js/main.js

```diff
@@ -1,2 +1,42 @@
+/* eslint-disable no-global-assign */
 /* global data */
 /* exported data */
+
+var $inputURL = document.querySelector('#url');
+var $img = document.querySelector('.square-img');
+
+$inputURL.addEventListener('input', function (event) {
+  $img.setAttribute('src', event.target.value);
+});
+
+var $title = document.querySelector('#title');
+var $textArea = document.querySelector('#textarea');
+var $form = document.querySelector('.form');
+
+var prevData = localStorage.getItem('jv-local-storage');
+
+if (prevData !== null) {
+  data = JSON.parse(prevData);
+}
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entryObj = {};
+  entryObj.title = $title.value;
+  entryObj.src = $img.getAttribute('src');
+  entryObj.textArea = $textArea.value;
```

Comment:
This is okay, but you should use the `elements` property of the `$form` DOM element.